docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -287,6 +281,7 @@ aws serverlessrepo list-application-versions --application-id arn:aws:serverless
             ElasticServerlessForwarderS3ConfigFile: ""
             ElasticServerlessForwarderSSMSecrets: ""
             ElasticServerlessForwarderKMSKeys: ""
+            ElasticServerlessForwarderAssumeRoles: ""
```

Comment:
this must be removed: it is not needed.
probably a leftover since you started the review and I pushed the latest change